### PR TITLE
Object detection

### PR DESCRIPTION
**Pull request by SAlexisvf**
Object Detection Model with TensorFlow API using Google COLAB

### CODE REVIEW

File: object_detection/.gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-venv/
\ No newline at end of file
+venv/
+*.DS_Store
```
Comment (diegocardozo97) [inline object_detection/.gitignore]: Would it be better to move this to the main .gitignore and adding tho them /object_detection? 
Like:  /object_detection/venv/,  /object_detection/*.DS_Store

**Comment by diegocardozo97** [comment]
lgtm, please check the comment about the .gitignore and maybe later we could check if there is a better way to upload the test/train images instead of in the main repo.

File: object_detection/.gitignore (hunk 1 above)
Comment (SAlexisvf) [reply object_detection/.gitignore]: Agree, I moved it to the main .gitignore

**Comment by paulvazbad** [comment]
>:( I was reviewing this

File: object_detection/scripts/generate_tfrecord.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    if row_label == 'Jumex_botella':
+        return 1
+    elif row_label == 'Jumex_cajita':
+        return 2
+    elif row_label == 'Coca_cola':
+        return 3
+    elif row_label == 'Agua_ciel':
+        return 4
+    else:
+        None
```
Comment (paulvazbad) [inline object_detection/scripts/generate_tfrecord.py]: This could be a dict

**Pull request merged**